Formula/qtfaststart.rb

```diff
@@ -3,7 +3,7 @@ class Qtfaststart < Formula
   homepage "https://libav.org/"
   url "https://libav.org/releases/libav-12.3.tar.gz"
   sha256 "115b659022dd387f662e26fbc5bc0cc14ec18daa100003ffd34f4da0479b272e"
-  license "LGPL-2.1"
+  license :public_domain
```

Comment:
Although the `libav` source code is mostly LGPLv2.1+, we only install a single file `tools/qt-faststart.c`, which is placed in the public domain:
```
/*
 * qt-faststart.c, v0.2
 * by Mike Melanson (melanson@pcisys.net)
 * This file is placed in the public domain. Use the program however you
 * see fit.
 *
```